src/polymer/elements/paper-item-v1-to-v2.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as dom5 from 'dom5';
+import stripIndent = require('strip-indent');
+
+import {registry} from '../../registry';
+import {HtmlRule} from '../../html/rule';
+import {ParsedHtmlDocument, Severity, Warning} from 'polymer-analyzer';
+import {FixableWarning} from '../../warning';
+
+class PaperItemV1ToV2 extends HtmlRule {
```

Comment:
Can this be just a static predicate in `content-to-slot-usages`?

Reply:
Wow, yeah, I should just do this...

Reply:
Hmm, it looks like it's already there, maybe I can drop all of the non-test content of this PR?